gluetool/glue.py

```diff
@@ -942,6 +942,13 @@ class Glue(Configurable):
                 'help': 'Print command-line that would re-run the gluetool session',
                 'action': 'store_true'
             },
+            ('j', 'json-file'): {
+                'help': """
+                        If set, all log messages (including ``VERBOSE``) are stored in this file
+                        in a form of JSON structures.
+                        """,
+                'default': None
```

Comment:
Is it required to mention default value in help message?

Reply:
and all the `exc_info[..]` stuff below is too much magical

Reply:
I think so, it looks like our Pylint checkers are not applied to `gluetool` itself, otherwise it should raise a warning at this point.